lora.cpp

```diff
@@ -61,7 +63,10 @@ static void lora::sendCallback(int32_t status)
   {
     DEBUG_SERIAL_LOG.println("Sending failed (callback)");
   }
-  sending = false;
+  DEBUG_SERIAL_LOG.println("Lora send complete, checking for downlinks"); 
+  // api.system.timer.create((RAK_TIMER_ID)UPLINK_PROCESS_TIMER, (RAK_TIMER_HANDLER)[](void *) {uplinkProcess = false; 
+  // DEBUG_SERIAL_LOG.println("End of uplink routine"); }, RAK_TIMER_ONESHOT);
+  // api.system.timer.start((RAK_TIMER_ID)UPLINK_PROCESS_TIMER, 5000, (void *)1);
```

Comment:
Normally, I recommend taking out any commented-out code. Is there a particular reason to leave this here?

Reply:
I have now removed this code, as the `delay` method improves uplink consistency.